citylearn/building.py

```diff
@@ -1346,6 +1430,23 @@ def estimate_observation_space_limits(self, include_all: bool = None, periodic_n
                     low_limit[key] = self.heating_device.efficiency
                     high_limit[key] = self.heating_device.efficiency
 
+            elif "charger" in key:
+                if self.ev_chargers is not None:
+                    for charger in self.ev_chargers:
+                        if key == f'charger_{charger.charger_id}_connected_state' or key == f'charger_{charger.charger_id}_incoming_state':
+                            low_limit[key] = 0
+                            high_limit[key] = 1
+                        elif 'ev_charger_state' in key:
+                            low_limit[key] = 0
+                            high_limit[key] = 1
+                        elif any(value in key for value in
+                                 ["ev_estimated_departure_time", "ev_estimated_arrival_time"]):
+                            low_limit[key] = 0
```

Comment:
0-24 is 25 hours. Confirm if it is 0-23 or 1-24. CityLearn uses 1-24 so it should match